databases/relational-databases.md

```diff
@@ -55,10 +55,59 @@ To understand, query, and insert information into a relational database, a techn
 - [SQL Fiddle](http://sqlfiddle.com)
 
 ### Lesson
+
```

Comment:
Please move new material after the rest of the text that was already in ### lesson.